src/player.h

```diff
@@ -5,6 +5,7 @@
 #include "item.h"
 #include "player_activity.h"
 #include "weighted_list.h"
+#include "morale.h"
```

Comment:
Not sure, I think it was an automatic addition by the CLion IDE. I'll got see if it's really needed and remove it if not.